src/FluentValidation/Validators/StringEnumValidator.cs

```diff
@@ -0,0 +1,35 @@
+﻿namespace FluentValidation.Validators
+{
+	using System;
+	using System.Linq;
+	using System.Reflection;
+	using FluentValidation.Internal;
+	using Resources;
+
+	public class StringEnumValidator : PropertyValidator
+	{
+		private readonly Type _enumType;
+
+		private readonly bool _caseSensitive;
+
+		public StringEnumValidator(Type enumType) : this(enumType, false)
+		{
+		}
+
+		public StringEnumValidator(Type enumType, bool caseSensitive) : base(new LanguageStringSource(nameof(StringEnumValidator)))

```

Comment:
Re-use the EnumValidator message with `new LanguageStringSource(nameof(EnumValidator))`